### PR TITLE
amend validation isNumber and Normalized Date

### PR DESCRIPTION
**Pull request by fedelocci**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/ui/common/attributes/AttributeHypeLongMonoTextSettings.js [hunk 1]
```diff
@@ -2,12 +2,19 @@ import React from 'react';
 import { Field } from 'redux-form';
 import { FormattedMessage } from 'react-intl';
 import { Row, Col } from 'patternfly-react';
+import { isUndefined } from 'lodash';
+import { isNumber } from '@entando/utils';
 import RenderTextInput from 'ui/common/form/RenderTextInput';
 import FormLabel from 'ui/common/form/FormLabel';
-import { isNumber } from '@entando/utils';
+
+const validateIsNumber = (value) => {
```
Comment (alessioscalici) [inline src/ui/common/attributes/AttributeHypeLongMonoTextSettings.js]: This is not needed anymore after merging https://github.com/entando/frontend-libraries/pull/6

File: src/ui/common/attributes/AttributesNumber.js [hunk 2]
```diff
@@ -2,10 +2,15 @@ import React from 'react';
 import { Field } from 'redux-form';
 import { FormattedMessage } from 'react-intl';
 import { Row, Col } from 'patternfly-react';
+import { isUndefined } from 'lodash';
 import { isNumber } from '@entando/utils';
 import RenderTextInput from 'ui/common/form/RenderTextInput';
 import FormLabel from 'ui/common/form/FormLabel';
 
+const validateIsNumber = (value) => {
```
Comment (alessioscalici) [inline src/ui/common/attributes/AttributesNumber.js]: This is not needed anymore after merging entando/frontend-libraries#6

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**